src/app/screens/Home/layout.js

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import { shape, arrayOf, string, func } from 'prop-types';
+import { push } from 'connected-react-router';
+import { useDispatch } from 'react-redux';
+import { UTIconButton, UTLabel, UTTooltip } from '@widergy/energy-ui';
+import i18 from 'i18next';
+
+import { RECORD } from 'constants/routes';
+
+import { ReactComponent as Icon } from '../../assets/record_icon.svg';
+
+import styles from './styles.module.scss';
+
+const Home = ({ valuesToMap, sentenceToDisplay, actualDisplay, buttonRenderer }) => {
+  const dispatch = useDispatch();
+
+  const goToRecordButtonTippyProps = () => ({
+    placement: 'top',
+    interactive: false,
+    arrow: false,
+    boundary: 'viewport',
+    appendTo: document.body
+  });
```

Comment:
You can move this to constants. Also it doesn't need to be a function, you can just export the object.
```suggestion
  const TOOLTIP_CONFIG = {
    placement: 'top',
    interactive: false,
    arrow: false,
    boundary: 'viewport',
    appendTo: document.body
  };
```

and then just `tippyProps={TOOLTIP_CONFIG}`

Reply:
done!